src/ask_order.rs

```diff
@@ -2,14 +2,17 @@ use crate::contract_info::require_version;
 use crate::error::ContractError;
 use crate::msg::MigrateMsg;
 use crate::version_info::get_version_info;
-use cosmwasm_std::{Addr, Coin, DepsMut, Storage, Uint128};
-use cosmwasm_storage::{bucket, bucket_read, Bucket, ReadonlyBucket};
+use cosmwasm_std::{Addr, Coin, DepsMut, Uint128};
+use cw_storage_plus::Map;
 use provwasm_std::ProvenanceQuery;
 use schemars::JsonSchema;
 use semver::Version;
 use serde::{Deserialize, Serialize};
 
 pub static NAMESPACE_ORDER_ASK: &[u8] = b"ask";
+pub const NAMESPACE_ORDER_ASK_V2: &str = "ask";
```

Comment:
The mixing of `NAMESPACE_ORDER_ASK_V2`, which is used to create a storage map that's suffixed with `_V1` is somewhat confusing.  Also, is the existing usage of `NAMESPACE_ORDER_ASK` entirely removed? If so, you could just redefine `NAMESPACE_ORDER_ASK` as a `&str`.

Reply:
my mistake, i need to remove the old one